sdk/platform/win32/src/az_win32.c

```diff
@@ -1,34 +1,22 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include <az_platform_internal.h>
+#include <az_platform.h>
+
+#define WIN32_LEAN_AND_MEAN
+#define NOMINMAX
+#include <windows.h>
```

Comment:
Can you please add a comment explaining these?

Reply:
These are pretty much common knowledge in the Windows programming world, these macros are needed to be defined before including `windows.h`. Yet despite being pretty widely used, there isn't a small MSDN page I could point to - the data is spread across MSDN here and there - https://docs.microsoft.com/en-us/search/?search=WIN32_LEAN_AND_MEAN . I'll see what I can do.

Reply:
Done, to an extent